ssv/spectest/tests/runner/consensus/post_finish.go

```diff
@@ -27,69 +25,107 @@ func PostFinish() tests.SpecTest {
 				Duty:   &testingutils.TestingSyncCommitteeContributionDuty,
 				Messages: append(
 					testingutils.SSVDecidingMsgsV(testingutils.TestSyncCommitteeContributionConsensusData, ks, types.BNRoleSyncCommitteeContribution),
+					// post consensus
```

Comment:
why is this needed?

Reply:
we want runner to finish (hence the test "PostFinish"). The runner finishes only after processing post consensus is done. In other tests we used `finishRunner`. I personally less like this option because we tend to inject a state that can never be in a real run.

Reply:
you are changing the test, something was wrong before it?

Reply:
yes the test was broken...